dist/Leaflet.fullscreen.js

```diff
@@ -94,6 +94,7 @@ L.Map.include({
         } else {
             L.DomUtil.removeClass(container, 'leaflet-fullscreen-on');
         }
+        this.invalidateSize();
```

Comment:
Can you remove the calls to `invalidateSize` in `_enablePseudoFullscreen` and `_disablePseudoFullscreen`. Those both call `_setFullscreen`, so they no longer need to invalidate the size themselves.